src/main/kotlin/camp/nextstep/edu/racingcar/interfaces/CommandLineInterface.kt

```diff
@@ -0,0 +1,32 @@
+package camp.nextstep.edu.racingcar.interfaces
+
+import camp.nextstep.edu.racingcar.racing.Car
+import camp.nextstep.edu.racingcar.racing.Track
+
+class CommandLineInterface {
+
+    fun readCarNames(): List<String> {
+        println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
+        return requireNotNull(readLine()?.split(","))
+    }
+
+    fun readMoveCount(): Int {
+        println("시도할 횟수는 몇 회인가요?")
+        return requireNotNull(readLine()?.toInt())
+    }
+
+    fun draw(car: Car, track: Track, to: Int) {
+        print("${car.name}\t: ")
+        val traces = track.traces()
+        for (i in 0..to) {
+            if (traces.next()) print("⎼ ")
+        }
+        println()
```

Comment:
```suggestion
    fun draw(line: LineView, until: Int) { // LineView 생성자는 LineView(car, track)
        println(line.traces(until))
```

`Line` 클래스를 만들어서 한 줄을 그려주는 책임을 맡겨볼까요? 테스트도 만들 수 있을것 같아요. 도전해 보시겠어요?